### DISCUSSION TITLE
Cobbler installation across network segments

### DISCUSSION THREAD

**Discussion by 1392273211** [cobbler/cobbler-web#341] [Q&A]
The server ip of cobbler is: 192.168.77.100 Client: 192.168.77.101 The test is that it can be installed, then the client is: 173.0.100.20 Cross-segment, the server and client are open in the three-layer network or vxlan network protocol, then does cobbler support cross-segment installation

**Comment by SchoolGuy** [comment]
I have set this up internally at my company for a good while now. To support a PXE installation and proxing the DHCP you will need a DHCP/BOOTP proxy and your clients cannot use unicast DHCP/BOOTP requests to the DHCP server. If both this criteria are met then it is just a matter of having a "fake" block for the network interface that your server is in (to allow listening to both the proxied and real network).
